web/public/locales/en.json

```diff
@@ -208,6 +208,7 @@
   "electricityImportedFrom": "<b>%1$s %%</b> of electricity available in %4$s <b>%2$s</b> is imported from %5$s <b>%3$s</b>",
   "emissionsImportedFrom": "<b>%1$s %%</b> of emissions from electricity available in %4$s <b>%2$s</b> are imported from %5$s <b>%3$s</b>",
   "ofCO2eqPerMinute": "of CO₂eq per minute",
```

Comment:
Should we delete all of these?

Reply:
Yes! I've removed all of them / translated some to the equivalent of "of CO2eq" in the respective language. @VIKTORVAV99 Could you confirm the Swedish is correct 🇸🇪 ?